README.md

```diff
@@ -1 +1,205 @@
-# Math-MAgicians (Microverse)
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+
+#Math-maigicians
+
+Create with the aim of mentioning one of the thematics of events that I like the most that is that of the video games and using all my knowledge in the right from the first module of the program microverse to give it form and functionality. 
+
+Will have future changes in the next days! 
+
+<a name="readme-top"></a>
+
+
+<div align="center">
+
+  <h3 id="title"><b>Math-magicians</b></h3>
+
+</div>
+
+# 📖 Web page for Gamescom event! <a name="about-project"></a>
+
+> This is the first project of the react series im cirruntly in the aim is to use the elements of the react framework to develop a simple calculator
+
```

Comment:
- [ ] This section should contain the name of the application which is `Math Magician` and below it should be the description I kindly suggest you move the description which you specified at the top of this section into this section properly 👍

Reply:
Good Job working some requirements but it would be nice to take seriously what the previous code reviewer requested you to change. A read me file should be professional at all times because its the first thing a different user that's looking at your app with go through. kindly make this changer to avoid wasting your review chances